facia-tool/public/js/modules/vars.js

```diff
@@ -1,5 +1,5 @@
 import CONST from 'constants/defaults';
-import isEqual from 'underscore';
+import _ from 'underscore';
```

Comment:
underscore named exports don't work very well. I ended up with `isEqual` being the underscore object, not the `isEqual` function, and `{isEqual}` is undefined